tests/test_command_reference.rb

```diff
@@ -42,44 +43,40 @@ def write(string)
     @input_file.flush
   end
 
+  def test_data_sanity
+    # Make sure the actual YAML in our library loads for various platforms
+    CommandReference.new('', '')
```

Comment:
This is testing that we can create a CommandReference instance (using the 'live' YAML files) that doesn't refer to any specific API or platform, without any errors being thrown.
The following lines are similarly creating CommandReference instances for various known APIs and platforms.

Basically, it's a sanity check for our YAML files, as if there's any syntax error in any of them, this should catch it.